automatic/etcd/etcd.nuspec

```diff
@@ -5,43 +5,49 @@
     <id>etcd</id>
     <version>3.5.0</version>
     <packageSourceUrl>https://github.com/chocolatey-community/chocolatey-coreteampackages/tree/master/automatic/etcd</packageSourceUrl>
-    <owners>chocolatey-community,Andrei Bejenaru,Robert Labrie</owners>
+    <owners>chocolatey-community,dgalbraith,Andrei Bejenaru,Robert Labrie</owners>
     <title>etcd (Install)</title>
-    <authors>CoreOS</authors>
-    <projectUrl>https://github.com/coreos/etcd</projectUrl>
+    <authors>etcd Authors</authors>
+    <projectUrl>https://etcd.io</projectUrl>
     <iconUrl>https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-coreteampackages@c681fe2d8274d648911c90a5cfa63b43e4663013/icons/etcd.png</iconUrl>
-    <copyright>(c) 2017 CoreOS, Inc</copyright>
-    <licenseUrl>https://github.com/coreos/etcd/blob/master/LICENSE</licenseUrl>
+    <copyright>(c) 2013-2021 etcd Authors</copyright>
```

Comment:
If the previous copyright from 2017 was CoreOS, Inc then the new copyright would seem to be wrong? Should it not be `2013-2017 CoreOS, Inc, 2018-2021 etcd Authors` ?

Reply:
@pauby that is actually a little unclear so a shared view on this would be great.

etcd was initially written by CoreOS with the initial publication being in 2013 as part of the CoreOS Container Linux project.  

CoreOS continued to maintain and publish etcd through to 2018 when they ceased as a stand-alone entity after having been [acquired by Redhat](https://www.redhat.com/en/about/press-releases/red-hat-acquire-coreos-expanding-its-kubernetes-and-containers-leadership).  At the end of 2018 etcd joined the [CNCF](https://cncf.io) as a [project in incubation](https://www.cncf.io/blog/2018/12/11/cncf-to-host-etcd) and it [graduated in 2020](https://www.cncf.io/announcements/2020/11/24/cloud-native-computing-foundation-announces-etcd-graduation).

Despite the various iterations the Github history goes back to the [initial commit](https://github.com/etcd-io/etcd/commit/20ca21a3f7122cf7caa91cb0e9b9c69be9279950) on June 7 2013.

While it has all the history the repository itself doesn't provide any real direction on the ownership and copyright assignment - while the software is licensed under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.html) there is no indication on the date range or copyright ownership in the LICENSE file which just represents the raw Apache 2.0 License text [committed on 31 July 2013](https://github.com/etcd-io/etcd/commit/3381177341c11aa9d89b6172699d64ddc2146a11).

The only real inference on a possible intent on the copyright assignment comes with the public web site at (https://etcd.io).  The copyright information on the web page (bottom bar of the display) shows `© 2013-2021 etcd Authors` and the [LICENSE file](https://github.com/etcd-io/website/blob/main/LICENSE) for the website indicates a `Copyright 2013-present etcd Authors. All rights reserved` but noting that this was only done in [March 2021](https://github.com/etcd-io/website/pull/122).

Given this it feels as though there are 2 options, both of which have seemingly equal merit:
i) Split the copyright attribution as per your suggestion; or
ii) Roll the entire copyright into a single attribution to etcd Authors coving the entire period

Just wanted to flag the background for both with you on the review - I am happy to go with either option.

Given the additional context here would you like me to move towards a split attribution like `2013-2017 CoreOS, Inc, 2018-2021 etcd Authors` ?

Reply:
> The copyright information on the web page (bottom bar of the display) shows © 2013-2021 etcd Authors and the LICENSE file for the website indicates a Copyright 2013-present etcd Authors. All rights reserved 

Given that they are asserting those dates and copyright we should follow it that way we're not best guessing it and using what we know and can point to.

Thanks for looking into this so deeply @dgalbraith !